setup.py

```diff
@@ -80,7 +80,7 @@ def package_vars(version_file):
             # coveralls does not currently support coverage 5.0
```

Comment:
So can we remove this comment now?  It looks like coveralls-clients/coveralls-python#203 was resolved some time ago.

Reply:
Yep, looks like those comment lines can be removed.

Reply:
Resolved in https://github.com/cisagov/development-guide/pull/28/commits/99dc30c5561d32ea4fea093b9954579a4e84b9ca